config/services.yaml

```
@@ -34,8 +48,44 @@ services:
     App\EventSubscriber\CommentNotificationSubscriber:
         $sender: '%app.notifications.email_sender%'
 
+    Symfony\Component\Finder\Finder:
+
+    App\EventListener\ApiContentTypeListener:
+        tags:
+            - {name: kernel.event_listener, event: kernel.request, priority: 33}
+
+    App\Service\Notification\TelegramNotificationChannel:
+        arguments:
+            - '%telegram_api_key%'
+            - '%telegram_send_message_uri%'
+            - '%telegram_chat_id%'
+        tags:
+            - { name: 'app.handler', key: 'handler_telegram' }
```

```suggestion
            - { name: 'notification_channel', key: 'telegram' }
```
- Keys can be simplified
- Tag name should represent it's purpose